3.5/aspnet/windowsservercore-1803/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# escape=`
+
+FROM mcr.microsoft.com/dotnet/framework/runtime:3.5-windowsservercore-1803
+
+RUN powershell -Command Add-WindowsFeature Web-Server & `
```

Comment:
Invoking powershell.exe for each cmd within the `RUN` instruction is not very efficient.  Please log a follow-up issue to address this.

Reply:
In addition, our normal convention for multi-command statements is to use double-ampersand `&&` which will ensure that the entire call fails if any of the individual commands fail.  By using a single ampersand `&` it will silently fail and continue to execute the next command.  And each line after the initial command should be preceded by `&&`.

Example:
```
RUN command1 `
    && command2 `
    && command3
```

Reply:
#333